### PR TITLE
Use cli.log arguments

### PR DESCRIPTION
**Pull request by max-hoffman**
`Dolt.cli.log()` was not implementing number or commit.

### CODE REVIEW

**Comment by max-hoffman** [comment]
@oscarbatori there are tests failing related to the commit parent code. I'll take another look in the morning but could use your look at code to see if the original tests were flaky or if i broke something.

**Comment by max-hoffman** [comment]
Nvm everything is sorted here, good for review.

File: tests/cli/test_dolt.py [hunk 1]
```diff
@@ -124,6 +124,7 @@ def test_merge_conflict(create_test_table):
     repo.merge('other', message_merge)
 
     commits = list(repo.log().values())
+    print(commits)
```
Comment (oscarbatori) [inline tests/cli/test_dolt.py:127]: logger?

File: tests/cli/test_dolt.py [hunk 2]
```diff
@@ -146,6 +147,41 @@ def test_dolt_log(create_test_table):
     assert previous_commit.message == message_one
 
 
+def test_dolt_log_number(create_test_table):
```
Comment (oscarbatori) [inline tests/cli/test_dolt.py:150]: DRY Dolt `test_dolt_log_*` with helper?

**Review by oscarbatori — APPROVED** [review]
LGTM, some optional remarks

File: tests/cli/test_dolt.py (hunk 2 above)
Comment (max-hoffman) [reply tests/cli/test_dolt.py:149]: I agree there's a more principled way to probably do this, I might want to do something like adding a .sql file that can create an appropriate setup state for most CLI+SQL tests at some point.

**Pull request merged**